SampleApp/AutocompleteTableVC.swift

```diff
@@ -27,21 +27,19 @@ class AutocompleteTableVC: UITableViewController, UISearchResultsUpdating, UISea
     self.definesPresentationContext = true
     searchController.searchBar.sizeToFit()
     manager.delegate = self
-    PeliasSearchManager.sharedInstance.urlQueryItems = [NSURLQueryItem(name: "apiKey", value: "mapzen-2qQR7SX")]
+    PeliasSearchManager.sharedInstance.urlQueryItems = [NSURLQueryItem(name: "api_key", value: "mapzen-2qQR7SX")]
```

Comment:
Plist or code-based is probably best. Now that we have a decent way todo it in Tangram-es we can have either/both.